frontend/packages/core/src/Theme/colors.tsx

```diff
@@ -173,6 +173,23 @@ export const DARK_COLORS: ClutchColors = {
   },
 };
 
+export const clutchChartColors: string[] = [
```

Comment:
Hmm, I wonder if we should be a little more descriptive with how we approach this. If we're to support multiple themes then this will likely not work well as it is very static. Also it should probably be a child of `colors` so it will inherit the variant that is passed down. Then also, what would we do if we wanted to hold different colors based on the chart/type etc?

Reply:
Adjusted the charts colors prop to be under `colors`, so it'll look like

```typescript
interface ChartColors {
  common: string[];
}

export interface ClutchColors {
  neutral: Color;
  blue: Color;
  green: Color;
  amber: Color;
  red: Color;
  charts: ChartColors;
}
``` 
`commons` felt right since the colors collection could be used by multiple charts. Having charts, with a different structure, within ClutchColors seems a bit odd though, wdyt?

Reply:
So one structure we could utilize is:

```
charts: {
  data: [],
  pie: {},
  graph: {},
}
```

This way we have the common `data` colors that all charts will likely use and then in the future we have the capability of subbing the type of charts with further data. 

`common` feels like it could work but the word intones that these are all common colors shared across and therefore would likely have children of its own, such as `common.data`.

So I think we can utilize what I mentioned above as:

```
charts: {
  common: {
    data: []
  },
  pie: {},
  graph: {},
}
```

This way it allows us to add further `common` variables in the future if we need to.

Reply:
Sadly I don't know 'what' we'll need in the future, so this is just preparation to make sure we can support it :)